### ISSUE TITLE
Uncaught TypeError (svelte, bundled with vite)

### ISSUE THREAD

**Issue by mustafa0x** [StackExchange/Stacks-Editor#139]
**Describe the bug**
Uncaught TypeError: Class constructor PluginKey cannot be invoked without 'new'

![image](https://user-images.githubusercontent.com/1473102/173246758-90c69f5e-0591-46e9-94f9-f873ba23a30c.png)

package.json
```
    "@stackoverflow/stacks-editor": "^0.5.0", // I also installed from main, but same error
    "@stackoverflow/stacks": "^1.1.0",
    "vite": "^2.9.9",
```

Svelte file
```svelte
<div bind:this={cont} />

<script>
import { StacksEditor } from "@stackoverflow/stacks-editor"
// don't forget to include the styles as well
import "@stackoverflow/stacks-editor/dist/styles.css"
// include the Stacks js and css as they're not included in the bundle
import "@stackoverflow/stacks"
import "@stackoverflow/stacks/dist/css/stacks.css"

let cont
onMount(() => {
    new StacksEditor(cont, "*Your* **markdown** here")
})
</script>
```

Repro: https://stackblitz.com/edit/vitejs-vite-umrwjx

**Comment by mustafa0x** [comment]
Seems to be fixed in 0.5.0, thanks.

**Comment by b-kelly** [comment]
Sorry for the late response - this came in while I was on vacation and I totally missed it. I'm not sure what exactly the issue was, but I'll go back and check so I can make sure it doesn't happen again. I have a slight guess that it may have had to do with a mismatch in upstream types. ProseMirror switched over to using TypeScript (🎉), but didn't increment the major version number (as there was technically no backwards compat breaking in their api). This means that we _may_ have had some type mismatches during the window where ProseMirror was shipping one set of types and we were using a _different_ set of types from DefinitelyTyped.
